docs/API.md

```diff
@@ -1349,6 +1349,25 @@ Trying to add a batch when wrong *username* or *api key* is used:
 }
 ```
 
+Trying to add a batch with an empty list of domains:
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "error": {
+    "data": [
+      {
+        "message": "Not enough items: 0/1.",
```

Comment:
How should I interpret "0/1"?

Reply:
This means that there are 0 item out of 1 expected (this is defined [in the module](https://metacpan.org/release/JHTHORSEN/JSON-Validator-5.08/source/lib/JSON/Validator/Error.pm#L14)).

Reply:
A list of at list one item is expected. Is the list the missing item? For me it looks like we expect exactly 1 domain name.

Why not [contains] in the module?

[contains]: https://metacpan.org/release/JHTHORSEN/JSON-Validator-5.08/source/lib/JSON/Validator/Error.pm#L16

Reply:
Is the purpose of "Not enough items: 0/1." that it should go to a human reader? Or is "0/1" interpreted by a machine?

Again, should I interpret "1" as one list or one item?